server/routes/journeys/update-referral/description/controller.ts

```diff
@@ -0,0 +1,32 @@
+import { NextFunction, Request, Response } from 'express'
+import { SchemaType } from '../../referral/description/schemas'
+import { MESSAGE_REFERRAL_DETAILS_UPDATED, PatchReferralController } from '../../base/patchReferralController'
+import { generateSaveTimestamp, getMaxCharsAndThreshold } from '../../../../utils/appendFieldUtils'
+
+export class UpdateDescriptionController extends PatchReferralController {
+  GET = async (req: Request, res: Response) => {
+    res.render('referral/description/view', {
+      isProactiveReferral: req.journeyData.csipRecord!.referral.isProactiveReferral,
+      currentDescriptionOfConcern: req.journeyData.csipRecord!.referral.descriptionOfConcern,
+      descriptionOfConcern: res.locals.formResponses?.['descriptionOfConcern'],
+      isUpdate: true,
+      recordUuid: req.journeyData.csipRecord!.recordUuid,
+      ...getMaxCharsAndThreshold(req, req.journeyData.csipRecord!.referral.descriptionOfConcern),
+    })
+  }
+
+  checkSubmitToAPI = async (req: Request<unknown, unknown, SchemaType>, res: Response, next: NextFunction) =>
+    this.submitChanges({
+      req,
+      res,
+      next,
+      changes: {
+        descriptionOfConcern: generateSaveTimestamp(req.journeyData.prisoner!) + req.body.descriptionOfConcern,
+      },
```

Comment:
backend api doesn't append descriptionOfConcern. we have to append/concatenate the content here. 
so it's going to be `req.journeyData.referral!.descriptionOfConcern + generateSaveTimestamp(req.journeyData.prisoner!) + req.body.descriptionOfConcern,`